File: YouzanBasicSample/src/main/java/com/youzanyun/sdk/sample/basic/YouzanActivity.java
/*
 * Copyright (C) 2017 youzanyun.com, Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package com.youzanyun.sdk.sample.basic;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;


public class YouzanActivity extends AppCompatActivity {
    public static final String KEY_URL = "url";
    private YouzanFragment mFragment;


    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_placeholder);
        mFragment = new YouzanFragment();
        mFragment.setArguments(getIntent().getExtras());

        getSupportFragmentManager()
                .beginTransaction()
                .replace(R.id.placeholder, mFragment)
                .commitAllowingStateLoss();
    }

    @Override
    public void onBackPressed() {
        if (mFragment == null || !mFragment.onBackPressed()) {
            super.onBackPressed();
        }
    }
}

File: YouzanX5Sample/src/main/java/com/youzanyun/sdk/sample/x5/YouzanActivity.kt
/*
 * Copyright (C) 2017 youzanyun.com, Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package com.youzanyun.sdk.sample.x5

import android.app.AppOpsManager
import android.app.AsyncNotedAppOp
import android.app.SyncNotedAppOp
import android.net.wifi.WifiManager
import android.os.Build
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.util.Log

class YouzanActivity : AppCompatActivity() {
    private var mFragment: YouzanFragment? = null
    private val MY_APP_TAG = "MY_APP_TAG"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_placeholder)



        mFragment = YouzanFragment()
        mFragment!!.arguments = intent.extras
        supportFragmentManager
            .beginTransaction()
            .replace(R.id.placeholder, mFragment!!)
            .commitAllowingStateLoss()
    }


    override fun onResume() {
        super.onResume()
    }

    override fun onBackPressed() {
        if (mFragment == null || !mFragment!!.onBackPressed()) {
            super.onBackPressed()
        }
    }

    companion object {
        const val KEY_URL = "url"
    }
}